sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/implementation/models/KeyRotationPolicyAttributes.java

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.keyvault.keys.implementation.models;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * The key rotation policy attributes.
+ */
+@Fluent
+public final class KeyRotationPolicyAttributes {
+    @JsonProperty(value = "expiryTime")
+    private String expiryTime;
```

Comment:
What do other Java libraries use to model durations? we use string in JS because it's idiomatic for us, but Java might have a different way to model these types... consider checking with the [service bus](https://docs.microsoft.com/en-us/rest/api/servicebus/preview/queues/get#sbqueue) feature crew on how they model ISO8601 durations

Reply:
Eh feel free to ignore me if this is all internal / serialization / generated code... I really don't know Java 😄

Reply:
In the Java SDK we do have a preferred datetime type: `OffsetDateTime`. For serialization/deserialization internal classes can keep using `String` for `expiryTime` and `Long` for `createdOn` and `updatedOn`, but I think it would be better to use `OffsetDateTime` in public classes like you suggest.